src/mixins/map-mixin.js

```diff
@@ -644,7 +644,15 @@ export default function mapMixin(
     const layers = _map.getStyle().layers
     for (let i = 0; i < layers.length; ++i) {
       if (layers[i].type === "symbol") {
-        firstSymbolId = layers[i].id
+        // Streets and Outdoors styles are sets of layers thus only need to make the street label layer on top of omnisci layer
+        if (
+          layers[i].id === "tunnel-oneway-arrows-blue-major" ||
+          layers[i].id === "tunnel-oneway-arrows-blue-minor"
```

Comment:
It feels confusing to use these two layer IDs to set the `firstSymbolId`. Is there no way to check if the map type `=== "Streets" || === "Outdoors"`?

Reply:
Well there is a comment above to make it clear. Also, there is no information about the "Streets" and "Outdoors" within the layer in the layers array from the Mapbox. 
<img width="522" alt="Screen Shot 2019-10-28 at 9 59 28 AM" src="https://user-images.githubusercontent.com/11098472/67699770-e75c9f00-f969-11e9-8510-5eb3e26247a9.png">

Reply:
`_map.getStyle()` returns the name of the map:

```
{
  "version": 8,
  "name": "Mapbox Outdoors",
  "metadata": {
    "mapbox:autocomposite": true,
    "mapbox:type": "default",
    "mapbox:origin": "outdoors-v10",
    "mapbox:groups": {
      "1444934828655.3389": { "name": "Aeroways", "collapsed": true },
      "1444933322393.2852": {
        "name": "POI labels  (scalerank 1)",
        "collapsed": true
      },
```

Reply:
@CaitW, made a change to use style name if it looks better ;)